models/migrations/v27.go

```diff
@@ -18,18 +21,10 @@ func convertIntervalToDuration(x *xorm.Engine) (err error) {
 		Name    string
 	}
 	type Mirror struct {
-		ID          int64       `xorm:"pk autoincr"`
-		RepoID      int64       `xorm:"INDEX"`
-		Repo        *Repository `xorm:"-"`
-		Interval    time.Duration
-		EnablePrune bool `xorm:"NOT NULL DEFAULT true"`
-
-		Updated        time.Time `xorm:"-"`
-		UpdatedUnix    int64     `xorm:"INDEX"`
-		NextUpdate     time.Time `xorm:"-"`
-		NextUpdateUnix int64     `xorm:"INDEX"`
-
-		address string `xorm:"-"`
+		ID       int64       `xorm:"pk autoincr"`
+		RepoID   int64       `xorm:"INDEX"`
+		Repo     *Repository `xorm:"-"`
+		Interval time.Duration
```

Comment:
No, just don't need them during the migration. As I understand it Xorm will notice and warn but fields are still there. Same as repo struct above. It has more fields not listed.